src/index.js

```diff
@@ -0,0 +1,145 @@
+
+const tempValue = document.getElementById('tempValue');
+const increaseTempControl = document.getElementById('increaseTempControl');
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+const landScape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const getRealTimeTempButton = document.getElementById('getRealTimeTempButton');
+
+//Update the sky options when we choose different sky and change the color background in garden
+const skyOptions = {
+    'sunny': {
+        'text': "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+        'backgroundColor': "#DFFFFF",
+    },
+    'cloudy': {
+        'text': "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+        'backgroundColor': "#D3D3D3",
+    },
+    'rainy': {
+        'text': "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+        'backgroundColor': "#B0C3DF",
+    },
+    'snowy': {
+        'text': "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+        'backgroundColor': "#ADD9E6",
+    },
+};
+const skySelect = document.getElementById("skySelect");
+const skyDisplay = document.getElementById("skyDisplay")
+const gardenContent = document.getElementById("gardenContent")
+
+skySelect.addEventListener("change", (event) => {
+    const skySelected = event.target.value;
+    console.log("Selected sky:", skySelected);
+    updateSky(skySelected);
+});
+
+const updateSky = (sky) => {
+    const skySelectOption = skyOptions[sky]
+    console.log(skySelectOption)
+    if (skySelectOption) {
+        skyDisplay.textContent = skySelectOption.text;
+        gardenContent.style.backgroundColor = skySelectOption.backgroundColor;
+    }    
+};
+
+//Update the temperature and change font color and landscape when increase and decrease temperature
+const updateTemperature = () => {
+    tempValue.textContent = `${temperature}°F`;
+    tempValue.style.color = getFontColor(temperature);
+    landScape.textContent = getLandscape(temperature);
+};
+
+const getLandscape = (temp) => {
+    if (temp <= 59) return '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    if (temp <= 69) return '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    if (temp <= 79) return '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    return '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+};
+
+const getFontColor = (temp) => {
+    if (temp <= 49) return 'teal';
+    if (temp <= 59) return 'green';
+    if (temp <= 69) return 'yellow';
+    if (temp <= 79) return 'orange';
+    return 'red';
+};
+
+increaseTempControl.addEventListener('click', () => {
+    temperature += 1;
+    updateTemperature();
+});
+
+decreaseTempControl.addEventListener('click', () => {
+    temperature -= 1;
+    updateTemperature();
+});
+updateTemperature();
+
+//getRealTimeTemp button updates the temperature when we update the city
+getRealTimeTempButton.addEventListener('click', () => {
+    const city = headerCityName.textContent;
+
+    getCityLocation (city)
+    .then(({ lat, lon }) => {
+        return getCityTemperature(lat,lon);
```

Comment:
I don't think you actually need to return this since you aren't chaining it to anything else.